bodhi/models/models.py

```diff
@@ -706,16 +707,6 @@ def new(cls, request, data):
                     build.package for build in data['builds']
                 ] if pkg.requirements], []))))
 
-        if not data['autokarma']:
-            del(data['stable_karma'])
-            del(data['unstable_karma'])
-            caveats.append({
-                'name': 'autokarma',
-                'description': 'Auto-karma stable requests disabled.',
-            })
-
-        del(data['autokarma'])
```

Comment:
Everything looks good to me, nice work!! 

I think the next step will be for me to merge this PR, spin up a new release, and get it into staging for further testing.